src/main/java/org/jenkinsci/plugins/android_lint/LintPublisher.java

```diff
@@ -18,20 +16,29 @@
 import org.apache.commons.lang.StringUtils;
 import org.jenkinsci.plugins.android_lint.parser.LintParser;
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
 
-/** Publishes the results of parsing an Android lint file. */
+/**
+ * Publishes the results of parsing an Android lint file.
+ */
 public class LintPublisher extends HealthAwarePublisher {
 
-    /** Plugin name used in console output. */
+    private static final long serialVersionUID = 3435696173660003622L;
+
+    /**
+     * Plugin name used in console output.
+     */
     private static final String PLUGIN_NAME = "android-lint";
 
-    /** Default filename pattern. */
+    /**
+     * Default filename pattern
+     */
     private static final String DEFAULT_PATTERN = "**/lint-results.xml";
 
-    private static final long serialVersionUID = 3435696173660003622L;
-
-    /** Ant fileset pattern of files to work with. */
-    private final String pattern;
+    /**
+     * Ant fileset pattern of files to work with.
+     */
+    private String pattern;
```

Comment:
> Back the original format

Thanks.

> I'd prefer my proposal

And I prefer the style which doesn't waste two lines of space for no reason whatsoever! :)